modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/presets/CKEditor5AdvancedPreset.ts

```diff
@@ -4,11 +4,82 @@
  */
 
 import {EditorConfig} from 'ckeditor5';
+import {sub} from 'frontend-js-web';
 
 const CKEditor5AdvancedPreset: EditorConfig = {
 	alignment: {
 		options: ['left', 'center', 'right'],
 	},
+	heading: {
```

Comment:
[nit] I don't think `CKEditor5*` prefix for presets in necessary. You know this from the dirs and components names. `AdvancedPreset` and `BasicPreset` should be enough.